hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/cli/RouterCLI.java

```diff
@@ -37,12 +66,48 @@ public RouterCLI(Configuration conf) {
     super(conf);
   }
 
+  private static void buildHelpMsg(String cmd, StringBuilder builder) {
+    UsageInfo usageInfo = ADMIN_USAGE.get(cmd);
+    if (usageInfo == null) {
+      return;
+    }
+
+    if (usageInfo.args != null) {
+      String space = (usageInfo.args == "") ? "" : " ";
+      builder.append("   " + cmd + space + usageInfo.args + ": "
+              + usageInfo.help);
+    } else {
+      builder.append("   " + cmd + ": " + usageInfo.help);
+    }
+  }
+
+  private static void buildIndividualUsageMsg(String cmd, StringBuilder builder) {
+    UsageInfo usageInfo = ADMIN_USAGE.get(cmd);
+    if (usageInfo == null) {
+      return;
+    }
+    if (usageInfo.args == null) {
+      builder.append("Usage: routeradmin [" + cmd + "]\n");
```

Comment:
If we have a StringBuilder, use append not +

Reply:
I will modify the code.